components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/internal/APIManagerComponent.java

```diff
@@ -932,6 +938,18 @@ private void configureNotificationEventPublisher() {
                 adapterParameters.put(APIConstants.PUBLISHING_MODE, APIConstants.NON_BLOCKING);
                 adapterParameters.put(APIConstants.PUBLISHING_TIME_OUT, "0");
                 adapterConfiguration.setStaticProperties(adapterParameters);
+                if (Boolean.parseBoolean(System.getenv("FEATURE_FLAG_REPLACE_EVENT_HUB"))) {
```

Comment:
shall we use a constant for the feature flag name as this is reusing in multiple places?

Reply:
Feature flag is a temporary property which will be removed after the feature is completely added to the product. This feature flag is already removed from the code.

Reply:
then its fine.